src/OnePlusBot/Base/EventHandlers.cs

```diff
@@ -22,6 +22,8 @@ public class CommandHandler
         private readonly CommandService _commands;
         private readonly IServiceProvider _services;
 
+        private static Regex messageRegex = new Regex("https://(canary.){0,1}discordapp.com/channels/(\\d+)/(\\d+)/(\\d+)", RegexOptions.Singleline | RegexOptions.Compiled);
```

Comment:
`{0,1}` could be re-written as `?`. It has the same meaning.
Also, since `canary` isn't being used, wouldn't it be better to mark it as a non-capturing group? `(?:canary.)`.